distribution.pm

```diff
@@ -212,6 +212,7 @@ sub script_output {
         my $heretag = 'EOT_' . $marker;
         my $cat     = "cat > $script_path << '$heretag'; echo $marker-\$?-";
         testapi::wait_serial($self->{serial_term_prompt}, undef, 0, no_regex => 1);
+        bmwqemu::log_call("Content of $script_path : $cat");
```

Comment:
I'd vote for some markers at least, or enclose the $cat inside double quotes or double brackets or something in that fashion?

Reply:
```suggestion
        bmwqemu::log_call("Content of $script_path : \n \"$cat\" \n ");
```

something like this ?

Reply:
Somehow, yes :)